src/models/endpoints/MessageRequest.ts

```diff
@@ -1,6 +1,8 @@
 import { Message } from "./Message";
 import { MessageNotes } from "./MessageNotes";
 
+export type ChatPrompt = "stable" | "nightly";
```

Comment:
add doc and need to export this in models/index.ts to resolve this warning: https://github.com/yext/chat-core/pull/19/files#diff-226325bc238099a9bf40ebfcb99320d62f9902eeab1b4ed764ad57b87cbed611R22